featuretests/package_test.go

```diff
@@ -51,3 +55,15 @@ func init() {
 func TestPackage(t *stdtesting.T) {
 	coretesting.MgoTestPackage(t)
 }
+
+func runCommand(c *gc.C, args ...string) (*cmd.Context, error) {
+	// Writers need to be reset, because
+	// they are set globally in the juju/cmd package and will
+	// return an error if we attempt to run two commands in the
+	// same test.
+	loggo.ResetWriters()
+	ctx, err := cmd.DefaultContext()
```

Comment:
can we use `coretesting.Context()` instead? then you can use `coretesting.Stderr(ctx)` where you're currently asserting bytes.Buffer